### PR TITLE
[COGNITION]: STEM Automated Decision - Consider adding an aria-describedby to the inside/outside US parentheticals

### PR DESCRIPTION
**Pull request by ksantiagoBAH**
## Description
As a screen reader user, I would like to know the correct number to call whether inside the US or outside the US. We have this text on the screen, but it's not semantically associated with the links. We could add an aria-describedby to remedy this.
https://app.zenhub.com/workspaces/vft-59c95ae5fda7577a9b3184f8/issues/department-of-veterans-affairs/va.gov-team/22599
## Testing done
Local testing with Apple voiceover

## Acceptance criteria
- [x] 0 axe browser plugin and/or CI violations reported
- [x] Screen readers announce the link, then "inside the US" or "outside the US" after a short pause

## Definition of done
- [x] Events are logged appropriately
- [x] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by shawkey-daniel — APPROVED** [review]
LGTM

**Review by delli-gatti-michael — APPROVED** [review]

**Comment by noahgelman** [comment]
The aria-describedby is an appropriate solution

@1Copenut this is a good example of another line of text that could be added to the global aria-describedby container we discussed a few weeks ago.

**Comment by 1Copenut** [comment]
This is solid, thanks y'all!

File: src/applications/claims-status/components/StemAskVAQuestions.jsx [hunk 1]
```diff
@@ -26,11 +26,26 @@ function StemAskVAQuestions() {
       <br />
       <p>Education Call Center:</p>
       <p>
-        <Telephone contact={CONTACTS.GI_BILL} /> (inside the U.S.)
+        <span
+          aria-describedby="inside-US-tele"
+          className="no-wrap"
+          href="tel:+18884424551"
```
Comment (Mottie) [inline src/applications/claims-status/components/StemAskVAQuestions.jsx:32]: I don't think this `href` has a purpose.

File: src/applications/claims-status/components/StemAskVAQuestions.jsx [hunk 2]
```diff
@@ -26,11 +26,26 @@ function StemAskVAQuestions() {
...
+          aria-label="8 8 8. 4 4 2. 4 5 5 1."
+        >
+          <Telephone contact={CONTACTS.GI_BILL} />
```
Comment (Mottie) [inline src/applications/claims-status/components/StemAskVAQuestions.jsx:35]: I'll look into updating the `<Telephone>` component to accept attributes like an `id` and `aria-describedby`.

**Review by Mottie — APPROVED** [review]

**Review by LinnJS — APPROVED** [review]

**Pull request merged**